experimental/traffic-portal/src/app/shared/theme-manager/theme-manager.service.ts

```diff
@@ -91,10 +91,12 @@ export class ThemeManagerService {
 	 * @param theme Theme to be stored
 	 */
 	private storeTheme(theme: Theme): void {
-		try {
-			window.localStorage.setItem(this.storageKey, JSON.stringify(theme));
-		} catch (e) {
-			console.error(`Unable to store theme into local storage: ${e}`);
+		if(typeof(window) !== "undefined") {
```

Comment:
We need to be injecting the `Window` token into this service. But you don't need to fix that in this PR.